File: src/main/java/project/Model/Board.java
package project.Model;

import java.util.Date;

public class Board {

	private int board_num;
	private String board_title;
	private Date board_date;
	private String board_content;
	private String board_name;

	public int getBoard_num() {
		return board_num;
	}

	public void setBoard_num(int board_num) {
		this.board_num = board_num;
	}

	public String getBoard_title() {
		return board_title;
	}

	public void setBoard_title(String board_title) {
		this.board_title = board_title;
	}

	public Date getBoard_date() {
		return board_date;
	}

	public void setBoard_date(Date board_date) {
		this.board_date = board_date;
	}

	public String getBoard_content() {
		return board_content;
	}

	public void setBoard_content(String board_content) {
		this.board_content = board_content;
	}

	public String getBoard_name() {
		return board_name;
	}

	public void setBoard_name(String board_name) {
		this.board_name = board_name;
	}

}

File: src/main/webapp/js/board.js
function board_check(){
	
	if($("#board_name").val()==""){
		alert("작성자를 입력하세요.");
		$("#board_name").focus();
		return false;
	}
	if($("#board_title").val()==""){
		alert("제목을 입력하세요.");
		$("#board_title").focus();
		return false;
	}
	if($("#board_date").val()==""){
		alert("작성일을 입력하세요.");
		$("#board_date").focus();
		return false;
	}
	if($("#board_content").val()==""){
		alert("내용을 입력하세요.");
		$("#board_content").focus();
		return false;
	}						
}